src/test/java/com/notably/logic/suggestion/SuggestionTestUtil.java

```diff
@@ -0,0 +1,109 @@
+import com.notably.model.viewstate.ViewStateModel;
+import com.notably.model.viewstate.ViewStateModelImpl;
+import com.notably.testutil.TypicalBlockModel;
+
+/**
+ * Contains helper methods for testing suggestion feature.
+ */
+public class SuggestionTestUtil {
+    private static Model model;
+
+    public static Model getModel() {
+        BlockModel blockModel = TypicalBlockModel.getTypicalBlockModel();
+        SuggestionModel suggestionModel = new SuggestionModelImpl();
+        ViewStateModel viewStateModel = new ViewStateModelImpl();
+        model = new ModelManager(blockModel, suggestionModel, viewStateModel);
+
+        return model;
+    }
+
+    public static AbsolutePath getToCs2103t() {
+        return TypicalBlockModel.PATH_TO_CS2103T;
+    }
+
+    public static List<SuggestionItem> getExpectedSuggestionsToCs2103t() {
+        SuggestionItem cs2103t = new SuggestionItemImpl(
+                TypicalBlockModel.PATH_TO_CS2103T.getStringRepresentation(), null);
+        SuggestionItem cs2103tLect = new SuggestionItemImpl(
+                TypicalBlockModel.PATH_TO_CS2103T_LECTURES.getStringRepresentation(), null);
+        SuggestionItem cs2103tTut = new SuggestionItemImpl(
+                TypicalBlockModel.PATH_TO_CS2103T_TUTORIALS.getStringRepresentation(), null);
+        SuggestionItem cs2103tTut1 = new SuggestionItemImpl(
+                TypicalBlockModel.PATH_TO_CS2103T_TUTORIAL_1.getStringRepresentation(), null);
+        SuggestionItem cs2103tTut2 = new SuggestionItemImpl(
+                TypicalBlockModel.PATH_TO_CS2103T_TUTORIAL_2.getStringRepresentation(), null);
+
+        List<SuggestionItem> expectedSuggestions = new ArrayList<>();
+        expectedSuggestions.add(cs2103t);
+        expectedSuggestions.add(cs2103tLect);
+        expectedSuggestions.add(cs2103tTut);
+        expectedSuggestions.add(cs2103tTut1);
+        expectedSuggestions.add(cs2103tTut2);
+
+        return expectedSuggestions;
+    }
+
+    public static List<String> getExpectedInputsToCs2103t(String commandWord, boolean hasPrefixTitle) {
+        List<String> expectedInputs = new ArrayList<>();
+        String prefix = "";
+
+        if (hasPrefixTitle) {
+            prefix = "-t ";
+        }
+
+        expectedInputs.add(commandWord + " " + prefix + TypicalBlockModel.PATH_TO_CS2103T.getStringRepresentation());
+        expectedInputs.add(commandWord + " " + prefix
+                + TypicalBlockModel.PATH_TO_CS2103T_LECTURES.getStringRepresentation());
+        expectedInputs.add(commandWord + " " + prefix
+                + TypicalBlockModel.PATH_TO_CS2103T_TUTORIALS.getStringRepresentation());
+        expectedInputs.add(commandWord + " " + prefix
+                + TypicalBlockModel.PATH_TO_CS2103T_TUTORIAL_1.getStringRepresentation());
+        expectedInputs.add(commandWord + " " + prefix
+                + TypicalBlockModel.PATH_TO_CS2103T_TUTORIAL_2.getStringRepresentation());
+
+        return expectedInputs;
+    }
```

Comment:
TBH, this method seems kind of flaky to me. The idea that you're hardcoding the number of whitespaces that separates `commandWord` from `prefix` puts me off.  It seems to result in a brittle test, since you can never test the case when the whitespaces are more than one. For example, you can never test for the case when the user input is "new&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-t&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;This is the title"

Also, how about renaming the method name?

Reply:
Thanks for pointing it out! I have changed it such that this method will take in the part of the `userInput` before the path itself.